docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -1078,6 +1078,112 @@ public @interface CanWrite {
 ----
 <1> Any method or class annotated with the `@CanWrite` annotation is secured with this `@PermissionsAllowed` annotation instance.
 
+[[permission-bean-params]]
+=== Pass `@BeanParam` parameters into a custom permission
+
+The `@PermissionsAllowed` annotation attribute `params` can refer to methods or fields of the secured method parameter.
```

Comment:
Yes, the idea is accessors.  If field is called `myField` then public method `myField` (because of records) and `getMyField` must be matched. I don't want to support EL.